packages/integration-sdk/CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ and this project adheres to
   environments to check whether an `Error` is of a type that need not be alerted
   to deployment operators. All `Error`s other than those only an end user can
   resolve should be reported.
+- Made `ToMatchGraphObjectSchemaParams.schema` property optional in jest
+  matcher.
+
+### Fixed
+
+- Fixed `toMatchGraphObjectSchema` matcher for non-array `_class` values.
```

Comment:
I think we've not been consistent, but want to have these refer to the issue number when they "Fixes" or "Closes" one.

Reply:
Thanks for the suggestion! Done.